src/brick-int.h

```diff
@@ -207,7 +207,7 @@ struct pg_brick *pg_brick_decref(struct pg_brick *brick,
 int pg_brick_reset(struct pg_brick *brick, struct pg_error **errp);
 
 /* testing */
-uint32_t pg_brick_links_count_get(const struct pg_brick *brick,
+int pg_brick_links_count_get(const struct pg_brick *brick,
```

Comment:
this change brick and had nothing to do with coverage, as sure it should be in a different patch

Reply:
^ this is still valid
Especially for brick which is use everywhere in packetgraph, change in brick deserver a separate commit